ftw/publisher/sender/workflows/subscribers.py

```diff
@@ -11,13 +11,21 @@
 import pkg_resources
 
 
+FTW_SIMPLELAYOUT_AVAILABLE = False
 try:
     pkg_resources.get_distribution('ftw.simplelayout')
+    FTW_SIMPLELAYOUT_AVAILABLE = True
 except pkg_resources.DistributionNotFound:
-    FTW_SIMPLELAYOUT_AVAILABLE = False
-else:
+    pass
+
+try:
+    pkg_resources.get_distribution('ftw.contentpage')
     FTW_SIMPLELAYOUT_AVAILABLE = True
-    from ftw.publisher.core.adapters.ftw_simplelayout import is_sl_contentish
+except pkg_resources.DistributionNotFound:
+    pass
+
+if FTW_SIMPLELAYOUT_AVAILABLE:
+    from ftw.publisher.core.adapters.simplelayout_utils import is_sl_contentish
```

Comment:
I think you can always import `is_sl_contentish`, no matter wether `ftw.contentpage` or `ftw.simeplelayout` is available (thanks to your refactoring in f.p.core).
I think you can just move the import to top-level, remove all package availability checks, remove `FTW_SIMPLELAYOUT_AVAILABLE` and just call the `is_sl_contentish`.
If no simplelayoutish framework is installed, `is_sl_contentish` just wont do much. Right?